### PR TITLE
Make digits="apa" compliant with APA 7th edition

### PR DESCRIPTION
**Pull request by bwiernik**
APA now strongly recommends/requires exact p values unless p < .001.

### CODE REVIEW

File: R/format_p.R [hunk 1]
```diff
@@ -46,17 +46,7 @@ format_p <- function(p, stars = FALSE, stars_only = FALSE, name = "p", missing =
...
-            ifelse(p > .999, "> .999",
-              paste0("= ", format_value(p, 3))
-            )
-          )
-        )
-      )
-    )
+    digits <- 3
   } else if (is.character(digits) && grepl("^scientific", digits)) {
     digits <- tryCatch(
```
Comment (strengejacke) [inline R/format_p.R:51]: the `if` statement previously returned `text`, the formatted p-value. I think the revision you're suggesting work when you don't put the first if-statement into an if-else-chain, but just start in line 50 with 

`if (is.character(digits) && grepl("^scientific", digits)) {`

**Review by strengejacke — CHANGES_REQUESTED** [review]

File: R/format_p.R (hunk 1 above)
Comment (strengejacke) [reply R/format_p.R:51]: and simply `if (digits == "apa") digits <- 3`.

Comment (bwiernik) [reply R/format_p.R:53]: ah thanks, yeah, misread something there

**Comment by strengejacke** [comment]
looks good, thanks!

**Pull request merged**